crates/polars-ops/src/series/ops/round.rs

```diff
@@ -48,27 +97,34 @@ pub trait RoundSeries: SeriesSealed {
             let multiplier = 10i128.pow(decimal_delta);
             let threshold = multiplier / 2;
 
-            let ca = ca
-                .apply_values(|v| {
-                    // We use rounding=ROUND_HALF_EVEN
+            let res = match mode {
+                RoundMode::HalfToEven => ca.apply_values(|v| {
                     let rem = v % multiplier;
                     let is_v_floor_even = ((v - rem) / multiplier) % 2 == 0;
```

Comment:
Can you please avoid the extra division by doing a modulo with `2 * multiplier` and some extra logic? `i128` division is very very expensive.